programmers-coding-test/sangjun/sort/두배열의원소교체.js

```diff
@@ -0,0 +1,30 @@
+const rl = require("readline-sync");
+const change = rl.question("바꿀 횟수 :\n>> ").split(" ");
+const N = rl.question("배열속 숫자 갯수 :\n>> ").split(" ");
+
+const firstGraph = [];
+const firsetVector = rl.question("첫번쨰 배열 숫자 입력 :\n>> ").split(" ").map((el) => parseInt(el));
+firstGraph.push(firsetVector);
+
+const secondGraph = [];
+const secondVector = rl.question("두번째 배열 숫자 입력 :\n>> ").split(" ").map((el) => parseInt(el));
+secondGraph.push(secondVector);
+
+firsetVector.sort((a, b) => a - b);
+secondVector.sort((a, b) => b - a);
```

Comment:
readline-sync는 실제 코딩테스트에서 사용이 불가하니, 아래 것을 이용할 것.
readline-sync로 연습하다간 실제 코딩테스트에서 난항을 겪을 수 있으니 미리 적응하도록 하자.

코딩테스트에서 입력받는 변수는 그대로 이용하는 것이 좋음
ex ) change -> K

vector 배열 그대로 이용하면 되는데 왜 graph 배열에 넣어준거...????!?!?!?!?
```suggestion

const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const [N, K] = input[0].split(" ").map(Number);

const firstVector = input[1].split(" ").map(Number).sort((a, b) => a - b);
const secondVector = input[2].split(" ").map(Number).sort((a, b) => b - a);
```